submitted_models/coro_mike_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,72 @@
+<!--- This is a Markdown description of a robot model submitted for inclusion in the
+DARPA Subterranean Challenge Technology Repository -->
+# Coordinated Robotics - Mike
+This specifications.md file is a description and proof of virtual model validation for
+Coordinated Robotics Mike with Sensor Configuration 1. This robot may be launched using
+a `ign launch` command with the variable name `coro_mike_sensor_config_1`.
+
+## Description
+Mike is a four-wheel skid steer ground vehicle using bicycle wheels and with a central joint
+splitting the robot into left and right. Mike weighs ~50 kg and uses a motor for each wheel.
+Mike can carry and launch a drone from atop the sensor platform.
+
+## Usage Instructions
+The robot is controlled by the skid steer plugin and accepts a linear x velocity and a rotational z velocity.
+Sensors are published on the standard topics.
+
+## Usage Rights
+
+
+## Cost and Scale
+Mike has an estimated duplication cost of USD 8,600.
+Cost breakdown:
+$2500 - Velodyne VLP-16 https://www.ebay.com/sch/i.html?_nkw=velodyne+vlp-16&_sacat=0
+$1800 - Frame / motors / wheels / batteries / controllers
+$1300 - MS-IMU3025
+$800  - lights / connectors /switches / DC-DC / misc
+$800  - Computer
+$225  - Seek Thermal Compact
+$200  - D435
+
+## Sensors
```

Comment:
Please include the Seek Thermal camera in this section as well.

Reply:
I added the thermal sensor to the list of sensors.